client/routes/index.js

```diff
@@ -38,6 +39,11 @@ export default () => (
       }
     />
     <Route exact path="/test-queue" element={<TestQueue />} />
+    <Route
```

Comment:
Wondering if this should be a page that can be viewed by "anonymous" (not signed in) users. Given how the live results of testers aren't visible to anonymous users, I figure this may page may also have no relevance to them.

Reply:
Good point. Changes here [0369fa0](https://github.com/w3c/aria-at-app/pull/1195/commits/0369fa0a6620388230d79f19dbe57d49f9dbb5ee)